robottelo/constants.py

```diff
@@ -48,10 +48,12 @@
 
 OPERATING_SYSTEMS = [
```

Comment:
Is this list the same as provided by NailGun?

If yes what about doing something like `OPERATING_SYSTEMS = nailgun.OSES`